frontend/gatsby-config.js

```diff
@@ -32,6 +32,21 @@ module.exports = {
         background_color: "#663399",
         theme_color: "#663399",
         display: "minimal-ui",
+
+        icon: `src/images/my-icon.png`,
+        icons: [
+          {
+            src: `src/images/my-icon.png`,
+            sizes: `48x48`,
+            type: `image/png`,
+          },
+          {
+            src: `src/images/my-icon.png`,
+            sizes: `72x72`,
+            type: `image/png`,
+          },
+        ],
```

Comment:
Hi @frames75 thanks for your PR.  I think you can remove this array and we can use automatic mode https://www.gatsbyjs.com/plugins/gatsby-plugin-manifest/#automatic-mode-configuration.

You'll need to add the image in `src/images`.  Could you change the name to favicon.png pr default-icon.png

For reference you can see how it is setup on: https://github.com/strapi/strapi-starter-gatsby-catalog/blob/main/frontend/gatsby-config.js

Reply:
That's perfect. 

I changed the *icon* parameter to `icon: 'src/images/favicon.png'` and added the `src/images/favicon.png`, obtained from the *strapi-starter-gatsby-catalog*.